snapshots/overlay/overlay.go

```diff
@@ -287,41 +287,37 @@ func (o *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 	if err != nil {
 		return nil, err
 	}
+	isRollback := true
+	defer func() {
+		if isRollback {
+			if rerr := t.Rollback(); rerr != nil {
+				log.G(ctx).WithError(rerr).Warn("Failure rolling back transaction")
```

Comment:
Can you update this error message to `failed to rollback transaction`

Reply:
The pr has been merged. I will modify it in the other pr later.